hknweb/models.py

```diff
@@ -9,6 +9,11 @@
 MAX_STRLEN = 85 # default max length for char fields
 MAX_TXTLEN = 2000 # default max length for text fields
 
+from django.utils import timezone
+
+MAX_STRLEN = 85
+MAX_TXTLEN = 2000
```

Comment:
These are already defined just above to the same value, so I don't think these are needed here (same with the timezone import, that's also already done just before this)